lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -35,150 +36,110 @@
 | Preparation
 |
 @preparation{
-- The kinesthetic activity in this lesson requires a ruler and 4-8 pennies for each group of 3. 
+- During the kinesthetic activity in this lesson, you will split your students into groups of 3. Each group will need a ruler, a marker, tape, and 5 pennies.
 - There is an optional activity in this lesson that would require you to print an extra data cycle for your students.
-- In each of the first 3 Investigate sections of this lesson, students will calculate a Measure of Center from a list. If you would like to be able to direct students to printed versions with space for them to write, make copies of @opt-printable-exercise{mean-median-modes.adoc}.
-- If you are using our Google Slides, adjust them based on which of these activities you will be doing with your students.
+- The first section of the lesson is an introduction to measures of center. If you are confident that your students have a solid understanding of mean, median, and mode, you can jump to the second section of the lesson, where students will use @proglang to compute measures of center, and then consider which measure of center is the most appropriate.
+- If you are using our Google Slides, adjust the slides based on which portions of the lesson you will be doing with your students.
 }
 
 |===
 
-== Mean @duration{15 minutes}
+== Introduction to Measures of Center @duration{15 minutes}
 
 === Overview
-Students learn about @vocab{mean} (or "average") as one way (among others!) to summarize a @vocab{quantitative} column, and how to compute it using @proglang.
+
+Students learn how to compute three different measures of center.
 
 === Launch
 
-One of the ways that Data Scientists summarize quantitative data is by talking about its _center_ - literally asking "what is a typical value in this sample?", in the hopes of inferring something about a larger population.  
+@teacher{Using the prompt below, invite your students to share what they know about mean, median, and mode. If your students need to review these concepts, continue to teach the entirety of this lesson section to provide instruction and practice. If your students are confident with computing mean, median, and mode, skip to the next section of the lesson, where students will (1) use Pyret to compute various measures of center and (2) begin considering the usefulness of each measure of center.}
 
-But there are many different ways to define "center", and each method has strengths and weaknesses. The shape of the data can play a huge role in whether or not one kind of summary is appropriate!
 
-@slidebreak
+In our work with dot plots and histograms, we often asked ourselves, "What is a typical value in this sample?"
```

Comment:
Hm...

I don't agree with this suggestion, because this is not a prompt. After reading the dot plot lesson, do you stand by this suggestion? (The dot plot lesson will precede this lesson, of course.) If not, perhaps I need to emphasize these ideas more during the dot plots lesson.